forest/benchmarking/tests/test_superoperator_tools.py

```diff
@@ -1,148 +1,149 @@
 import numpy as np
-from forest.benchmarking.utils import *
+from pyquil.gate_matrices import X, Z, H, relaxation_operators
 from forest.benchmarking.superoperator_tools import *
 
+
 # Test philosophy:
 # Using the by hand calculations found in the docs we check conversion
 # between one qubit channels with one Kraus operator (Hadamard) and two
 # Kraus operators (the amplitude damping channel). Additionally we check
 # a few two qubit channel conversions to get additional confidence.
 
-# The Amplitude Damping channel (one qubit)
```

Comment:
Hmm Im in superposition about the removal here. Here are my thoughts. 

1) pyQuil noise models are in flux now. We don't know what will happen.
2) this is self contained within forest.benchmarking so keeping it makes it easy to check with conventions in the markdown file.
3) the name relaxation does not match what we have in the docs and the rest of the code.
4) I get the flip side too. Reducing code duplication etc.

I guess in the end my superposition is slightly biased towards restoring the function but I'll leave it up to you.